### ISSUE TITLE
Is this really online?

### ISSUE THREAD

**Issue by wasd66666** [YaoZhang93/TumorCP#2]
Is this really a DA online? I see that "extract_tumor.py" saves the DA endwith "_tumor.npy" offline.

**Comment by YaoZhang93** [comment]
Yes but we did extract the tumor regions in advance to save time during training.

**Comment by wasd66666** [comment]
ok,thanks!
